ci-operator/step-registry/cucushift/upgrade/check-not-recommended/cucushift-upgrade-check-not-recommended-commands.sh

```diff
@@ -39,6 +39,13 @@ while (( retry > 0 )); do
         sleep 60
         echo "No conditionalUpdates update available! Retry..."
     else
+        #shellcheck disable=SC2076
+        if [[ " $unrecommended_conditional_updates " == *"failure determine thanos IP"* ]]; then
+            echo "Warning: Thanos IP is not ready, clearing CVO cache"
+            oc delete pod -n openshift-cluster-version -l app=cluster-version-operator
+            sleep 60
```

Comment:
shall we check if cvo pod restart successfully and re-get $unrecommended_conditional_updates?

Reply:
Updated. PTAL. Thanks!